core/domain/suggestion_registry.py

```diff
@@ -665,11 +672,12 @@ def __init__(
         )
         self.score_category = score_category
         self.language_code = language_code
-        # In BaseSuggestion, last_updated is defined with only datetime type
-        # but here last_updated is of Optional[datetime] type, because here
-        # last_updated can accept None values as well. So, due to this conflict
-        # in types MyPy throws an `Incompatible types in assignment` error.
-        # Thus to avoid the error, we used ignore here.
+        # Here we use MyPy ignore because in BaseSuggestion, last_updated
+        # is defined with only datetime type but here last_updated is of
+        # Optional[datetime] type because last_updated can accept None
```

Comment:
Same question as above, why can last_updated be None?

Reply:
https://github.com/oppia/oppia/pull/15905#discussion_r966848063 -- mentioned in this comment!

Reply:
Updated!